src/actor/ActorTemplate.js

```diff
@@ -205,34 +255,42 @@ exports = function(inherits) {
     }
 
     /**
-      * Set {@link Actor#vx} and {@link Actor#vy} to aim for the specified point, with the specified speed.
-      * @func Actor#headToward
-      * @arg {number} x
-      * @arg {number} y
-      * @arg {number} speed
-      */
+     * Set vx and vy to aim for the specified point, with the specified speed.
+     * @method ActorTemplate#headToward
+     * @param  {number} x
+     * @param  {number} y
+     * @param  {number} speed
+     */
     this.headToward = function(x, y, speed) {
       var targetAngle = Math.atan2(y - this.y, x - this.x);
       this.vx = speed * Math.cos(targetAngle);
       this.vy = speed * Math.sin(targetAngle);
     };
 
+    /**
+     * Rotate the actor at the specified point with the specified rotation offset
```

Comment:
"Rotate the actor at the specified point" is kind of vague and difficult to understand. Perhaps rotate the specified actor to point at?